config/scootconfig/scheduler_config_modules.go

```diff
@@ -0,0 +1,33 @@
+package scootconfig
```

Comment:
so the config section is actually called "SchedulerConfig" that's why its named that